astropy/samp/hub.py

```diff
@@ -1003,72 +1120,97 @@ def _notify_all_(self, sender_private_key, message):
                     if key != sender_private_key:
                         _recipient_id = self._private_keys[key][0]
                         recipient_ids.append(_recipient_id)
-                        self._launch_thread(target=self._notify,
-                                         args=(sender_private_key,
-                                               _recipient_id, message)
-                                         )
+                        self._launch_thread(
+                            target=self._notify,
+                            args=(sender_private_key, _recipient_id, message),
+                        )
 
         return recipient_ids
 
     def _call(self, private_key, recipient_id, msg_tag, message):
         self._update_last_activity_time(private_key)
 
         if private_key in self._private_keys:
-            if self._is_subscribed(self._public_id_to_private_key(recipient_id),
-                                   message["samp.mtype"]) is False:
-                raise SAMPProxyError(2, "Client {} not subscribed to MType {}"
-                                     .format(recipient_id, message["samp.mtype"]))
+            if (
+                self._is_subscribed(
+                    self._public_id_to_private_key(recipient_id), message["samp.mtype"]
+                )
+                is False
```

Comment:
Another instance that should be changed to ``not ...``

Reply:
Changed in commit 716d079. Feel free to revert it.